edgelet/contrib/config/linux/template.toml

```diff
@@ -40,6 +40,19 @@
 # manifest_trust_bundle_cert = "file:///var/secrets/manifest-trust-bundle.pem"
 
 
+# ==============================================================================
+# Elevated Docker Permissions Flag
+# ==============================================================================
+#
+# Some docker capabilities capabilities can be used to gain root access. 
+# By default, the --privileged flag and all capabilities listed in the CapAdd
+# field of the docker HostConfig will be ignored.
```

Comment:
This is backwards, right? They're not ignored by default because the config defaults to `true`.

Reply:
Ah yea, that's true for now, but we're planning to swap the default. I forgot we made the default true for now, I'll change the comment.

Reply:
Changed